app/dojo.py

```diff
@@ -1,13 +1,99 @@
-from person import Person, Fellow, Staff
-from room import OfficeSpace, LivingSpace
+from .person import Person, Fellow, Staff
+from .room import OfficeSpace, LivingSpace
+import random
+
+"""This file defines all functionalities for our office 
+space allocation app
+
+"""
 
 
 class Dojo(object):
+
 	def __init__(self):
-		pass
+		self.all_offices = []
+		self.all_livingspace = []
+		self.all_fellows = []
+		self.all_staff = []
+
+	def create_room(self, room_type, room_name):
+		"""Create a new room, either office or living space at the Dojo. 
+		 Check whether a room with a similar name exits,
+		 before succesfully creating the room
+
+		 """
+		if not isinstance (room_type,str) or  not isinstance (room_name, str):
+			raise ValueError("Use string values only")
+		if room_type.upper() == "OFFICE":
+			if room_name in [office.name for office in self.all_offices]:	
```

Comment:
Do you want your Dojo to contain same name for office and livingspace? I suggest you should check in both livingspace and offices. If there is a livingsapce called "Kenya" and a person wants to set up a new office called "Kenya" what will happen?